File: api/src/main/java/CC/CleanCircuit/entities/UserEntity.java
package CC.CleanCircuit.entities;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "tb_usuarios")
public class UserEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String nomeCompleto;
    @Column(unique = true, nullable = false)
    private String cpf;
    @Column(unique = true, nullable = false)
    private String email;
    @Column(nullable = false)
    private String senha;

    private String senhaTemporaria;

    private boolean senhaTemporariaBoolean;
}

File: CleanCircuitApp/app/src/main/java/com/guilhermematte/cleancircuitapp/data/model/entities/UserEntity.kt
package com.guilhermematte.cleancircuitapp.data.model.entities

data class UserEntity(
    val id: Int,
    val nomeCompleto: String,
    val cpf: String,
    val email: String,
    val senha: String,
    val date: String,
    val telefone: String,
    val urlProfileImage: String
)
